notebooks/MAST/HSC/CASJOBS/HSCV3_API/hscv3_api.ipynb

```diff
@@ -0,0 +1,636 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Hubble Source Catalog API Notebook\n",
+    "### August 2019, Rick White\n",
+    "#### July 2022 CI Updates, Trenton McKinney\n",
+    "\n",
+    "A [new MAST interface](https://catalogs.mast.stsci.edu/hsc) supports queries to the current and previous versions of the [Hubble Source Catalog](https://archive.stsci.edu/hst/hsc). It allows searches of the summary table (with multi-filter mean photometry) and the detailed table (with all the multi-epoch measurements).  It also has an associated [API](https://catalogs.mast.stsci.edu/docs/hsc.html), which is used in this notebook.\n",
+    "\n",
+    "This is based on [HSC Use Case #3](https://archive.stsci.edu/hst/hsc/help/use_case_3_v2.html).\n",
+    "* It searches the HSC for variable objects in the vicinity of dwarf galaxy IC 1613,\n",
+    "* shows the positions of those objects in a color-magnitude diagram,\n",
+    "* extracts light curves for an example object, and\n",
+    "* displays cutout images from the Hubble observations that were used for the light curve measurements.\n",
+    "\n",
+    "The whole process takes only 30 seconds to complete.\n",
+    "\n",
+    "Another [simple notebook](../HSCV3_SMC_API/hscv3_smc_api.ipynb) generates a color-magnitude diagram for the Small Magellanic Cloud in only a couple of minutes.  A more complex notebook that shows how to access the proper motion tables using the HSC API is also [available](../SWEEPS_HSCV3P1_API/sweeps_hscv3p1_api.ipynb)."
```

Comment:
```suggestion
    "Another [notebook](../HSCV3_SMC_API/hscv3_smc_api.ipynb) generates a color-magnitude diagram for the Small Magellanic Cloud in only a couple of minutes.  A more complex notebook that shows how to access the proper motion tables using the HSC API is also [available](../SWEEPS_HSCV3P1_API/sweeps_hscv3p1_api.ipynb)."
```
I think we should avoid words like "simple", since a reader might think to themselves "this isn't simple!" in the vein of textbooks that say "it is trivial to derive ..." when it's anything but trivial

Reply:
done